### PR TITLE
Add store load cairo cheatcodes

### LINKED ISSUE #1338: Add `store`, `load` cheatcodes

### PR DESCRIPTION
**Pull request by karol-bisztyga**
Closes #1338

### CODE REVIEW

File: protostar/starknet/cheaters/contracts.py [hunk 1]
```diff
@@ -288,3 +288,9 @@ async def call(
             general_config=StarknetGeneralConfig(),
         )
         return result.retdata
+
+    def store(self, name: str, value: Any) -> None:
+        self.cheatable_state.cairo_storage[name] = value
+
+    def load(self, name: str) -> Any:
+        return self.cheatable_state.cairo_storage.get(name)
```
Comment (karol-bisztyga) [inline protostar/starknet/cheaters/contracts.py:296]: TODO: Think about the type here. We could basically store anything but we can narrow it down to `CairoOrPythonData`, `str`, `int` or something

**Review by karol-bisztyga — COMMENTED** [review]
It kind of feels like this solution's too simple, LoL. Not sure if I missed anything or it's just what it is, not complicated.

**Comment by Arcticae** [comment]
This solution does not store values in contracts' storage, please refer to the old implementation for more details

File: tests/integration/pure_cairo_vm/cheatcodes/store_load/store_load_cairo_cheatcodes_test.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+
+async def test_store_load(
+    protostar: ProtostarFixture, run_cairo_test_runner: RunCairoTestRunnerFixture
+):
+    protostar.create_files(
+        {
+            "src/basic.cairo": CONTRACTS_PATH / "basic_contract.cairo",
+        }
+    )
```
Comment (karol-bisztyga) [inline tests/integration/pure_cairo_vm/cheatcodes/store_load/store_load_cairo_cheatcodes_test.py:29]: remove this

**Review by kasperski95 — APPROVED** [review]

**Pull request merged**